src/components/FilterTags/FilterTags.js

```diff
@@ -88,6 +98,15 @@ const FilterTags = () => {
     };
   }, []);
 
+  // Update query params when adding and removing filters
+  useEffect(() => {
+    const queryParams = queryString.parse(location.search);
+    queryParams.activeFilters = activeFilters;
+    history.push({
+      search: queryString.stringify(queryParams)
+    });
+  }, [activeFilters, history, location.search]);
```

Comment:
It's a bit weird here that the `location.search` is a 'read-dependency' but you're also setting it which should create an infinite loop, it doesn't though so that's good :) We also don't actually care if the history object changes here. The only variable we actually care about is the `activeFilters`. 

Instead of using `useEffect` here, it might be more appropriate to create a wrapper function around `setActiveFilters` which handles updating the query string like here: https://github.com/canonical-web-and-design/jaas-dashboard/blob/master/src/pages/Models/Models.js#L31